envs/docker.go

```diff
@@ -292,13 +296,16 @@ func (l *Local) dockerServiceToRelationship(client *docker.Client, container typ
 			username := ""
 			password := ""
 			path := ""
+			version := ""
 			for _, env := range c.Config.Env {
 				if strings.HasPrefix(env, "POSTGRES_USER") {
 					username = getEnvValue(env, "POSTGRES_USER")
 				} else if strings.HasPrefix(env, "POSTGRES_PASSWORD") {
 					password = getEnvValue(env, "POSTGRES_PASSWORD")
 				} else if strings.HasPrefix(env, "POSTGRES_DB") {
 					path = getEnvValue(env, "POSTGRES_DB")
+				} else if strings.HasPrefix(env, "PG_VERSION") {
```

Comment:
Is it something "standard"?

Reply:
This is the value used in the official postgresql image, see https://github.com/docker-library/postgres/blob/3db64a0e20b456a554b9da8791704cb3e54af7f2/Dockerfile-alpine.template#L20

Reply:
But nothing more official than this